apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/supervisor/client/impl/SvConnections.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright © 2020 Apollo Foundation
+ */
+package com.apollocurrency.aplwallet.apl.util.supervisor.client.impl;
+
+import com.apollocurrency.aplwallet.apl.util.supervisor.client.ConnectionStatus;
+import lombok.extern.slf4j.Slf4j;
+
+import java.net.URI;
+import java.util.Map;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Set of all connections inited
+ *
+ * @author alukin@gmail.com
+ */
+@Slf4j
+public class SvConnections {
+
```

Comment:
The class name is ambiguous, what about something like SvSession or else?

Reply:
renamed